libsolidity/experimental/analysis/TypeInference.cpp

```diff
@@ -676,7 +676,7 @@ bool TypeInference::visit(TypeClassInstantiation const& _typeClassInstantiation)
 		}
 	}
 
-	Type type{TypeConstant{*typeConstructor, arguments}};
+	Type targetType{TypeConstant{*typeConstructor, arguments}};
```

Comment:
By the way, I actually need a good name for this for an upcoming PR, where I need to make it annotation on `TypeClassInstantiation` in a new pass. I don't want to call it `type` because it would be confusing and suggest it's the type of the instantiation definition itself. Any suggestions? I came up with `targetType` or `instantiatedForType`, but neither seems particularly good. How is the type we're instantiating for typically referred to?

Reply:
I'll take `instanceType` then.

But is `typeInstance` even correct? Isn't this instantiation of the class?